cvat/apps/dataset_manager/formats/pascal_voc.py

```diff
@@ -56,6 +56,17 @@ def _import(src_file, instance_data, load_data_callback=None):
             for f in anno_files:
                 shutil.move(f, anno_dir)
 
+        # @update: add jpeg_dir @date: 2022/05/11
+        jpeg_dir = osp.join(tmp_dir, 'JPEGImages')
+        if not osp.isdir(jpeg_dir):
+            jpg_files = glob(osp.join(tmp_dir, '**', '*.jpg'), recursive=True)
+            jpeg_files = glob(osp.join(tmp_dir, '**', '*.jpeg'), recursive=True)
```

Comment:
Probably, I'd recommend to collect image files using [`these functions`](https://github.com/cvat-ai/datumaro/blob/v0.2/datumaro/util/image.py#L173-L207) from Datumaro. It will be much more generic than just collecting jpegs.

Reply:
Fixed